heartbeat/findif.sh

```diff
@@ -248,7 +250,7 @@ findif()
       fi
     fi
   else
-    if [ -z "$OCF_RESKEY_nic" -a "$netmask" != "${1#*/}" ] ; then
+    if [ -z "$OCF_RESKEY_nic" -a -z "$OCF_RESKEY_cidr_netmask" -a "$netmask" != "${1#*/}" ] ; then
```

Comment:
I'd recommend rewriting this as

```
if [ -z "$OCF_RESKEY_nic" ] && [ -z "$OCF_RESKEY_cidr_netmask" ] && [ "$netmask" != "${1#*/}" ] ; then
```

The `-a` operator doesn't short circuit and is not well-defined with more than 4 arguments, etc.

http://wiki.bash-hackers.org/commands/classictest#why_you_should_avoid_using_-a_and_-o

Reply:
Nice catch. I'll fix that.